solidity/contracts/TokenStakingEscrow.sol

```diff
@@ -247,6 +269,19 @@ contract TokenStakingEscrow is Ownable {
         emit DepositWithdrawn(operator, grantee, amount);
     }
 
+    function withdrawRevoked(
```

Comment:
Why we decided to implement two functions rather than put it into one?

Reply:
It's a different operation. `withdraw` and `withdrawToManagedGrantee` are called by grantee or grantee's operator to withdraw unlocked tokens to the grantee's account.

`withdrawRevoked` is a kind of a recovery function allowing grant manager to reclaim tokens from a revoked grant to do not lock them in the escrow. It's not a default flow, it's something called in exceptional cases. Those two operations are also separated in `TokenGrant` contract.

Reply:
I'm sorry I wasn't specific. What I mean are `withdrawRevoked` (public) and `withdrawRevoked` (internal).

Reply:
Oh, I thought it's just easier to follow. We have the standard case (`internal withdraw`) above the revoked case (`internal withdrawRevoked`) and they look pretty similar except that they calculate the amount differently, transfer tokens to different accounts, and fire different events. It's probably not that clear on a diff, but if you open the entire file and look at lines `266 - 290`, you'll see it then.

If you don't think it's better this way, I am happy to make it one function.

Reply:
We can leave it as is.